prime-angular-frontend/src/app/modules/enrolment/pages/regulatory/regulatory.component.ts

```diff
@@ -104,11 +109,23 @@ export class RegulatoryComponent extends BaseEnrolmentProfilePage implements OnI
     if (!this.certifications.length) {
       this.addEmptyCollegeCertification();
     }
+
+    const initialRemoteAccess = this.canRequestRemoteAccess();
+
+    this.form.valueChanges
```

Comment:
@johnnyl6 let me know if you have any questions, otherwise this PR is ready ![image](https://user-images.githubusercontent.com/599498/109375983-1e043b80-7876-11eb-93cf-9e920b4d6d0c.png)

Reply:
> @johnnyl6 let me know if you have any questions, otherwise this PR is ready ![image](https://user-images.githubusercontent.com/599498/109375983-1e043b80-7876-11eb-93cf-9e920b4d6d0c.png)

Good finding. LGTM✌